src/test/java/tc/DivisionTest.java

```diff
@@ -0,0 +1,33 @@
+package tc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.epam.tc.Operations;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import tc.DataProviders;
+
+public class DivisionTest {
+    private Operations operations;
+
+    @BeforeMethod(groups = {"DivMultiplyOperations"})
+    public void before() {
+        operations = new Operations();
+    }

```

Comment:
`@BeforeMethod` could be extracted to the some base class to avoid code duplication

Reply:
ok